models/Order.js

```diff
@@ -7,25 +7,47 @@ const orderSchema = new mongoose.Schema({
   status: {
     type: String,
     required: true,
-    enum: ['sent', 'pending', 'canceled'],
-    default: 'pending',
+    enum: ['CREATED', 'CONFIRMED', 'CANCELLED', 'REFUNDED', 'SENT', 'DELIVERED'],
+    default: 'CREATED',
   },
   user: {
     firstName: String,
     lastName: String,
+    patronymicName: String,
     email: String,
     phoneNumber: Number,
-    address: Address,
   },
   dateOfCreation: {
     type: Date,
     default: Date.now,
   },
+  lastUpdatedDate: Date,
+  completed: Boolean,
+  userComment: {
+    type: String,
+    default: '',
+  },
+  adminComment: {
+    type: String,
+    default: '',
+  },
+  cancellationReason: {
+    type: String,
+    default: '',
+  },
   delivery: {
     sentOn: Date,
     sentBy: String,
+    courier: Boolean,
+    courierOffice: Number,
```

Comment:
Where are we going to take value for this particular field?

Reply:
Number of courier office in the particular city/town will be provided by the client on checkout if he/she chooses Nova Poshta